uber/models/attendee.py

```diff
@@ -671,42 +676,60 @@ def is_transferable(self):
             not self.has_role_somewhere
 
     @property
-    def paid_for_a_swag_shirt(self):
+    def paid_for_a_shirt(self):
```

Comment:
The word "swag" was being used in the code a lot, and it ended up confusing me a lot.  Admittedly, this might just have been me being dense, but I couldn't stop thinking of ALL shirts as "swag" shirts.  Intuitively, it feels to me like if I show up at the merch booth and they hand me 3 shirts, then I have received 3 swag shirts, since swag is just cool stuff you're given.  However, the word was being used to refer specifically to the event shirts and NOT to the staff shirts.

I ended up removing the word "swag", and in places where we need to specify I replaced it with the word "event", which was clearer to me.

In this case I just removed it entirely, because it seemed redundant.